bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -158,15 +157,12 @@ object BagsState {
       (this, ioUnit)
 
     // Success, but now the observation has been edited so the AGS selection
-    // may not be valid.  Apply the results anyway in case the edit is not
-    // to a field that impacts AGS and go to pending to run again if necessary.
-    override def succeed(results: Option[AgsStrategy.Selection]): StateTransition = {
-      val action = for {
-        _ <- BagsManager.applyAction(obs, results)
-        _ <- BagsManager.wakeUpAction(obs, 0)
-      } yield ()
-      (PendingState(obs, Some(hash)), action)
-    }
+    // may not be valid. We could apply the results in case the edits made to the
+    // observation do not change the AGS selection, but in the interests of safety
+    // and as this was causing issues for BAGS overwriting PA edits, we only allow
+    // BAGS to apply to unedited observations.
+    override def succeed(results: Option[AgsStrategy.Selection]): StateTransition =
+      (PendingState(obs, Some(hash)), BagsManager.wakeUpAction(obs, 0))
```

Comment:
To make sure that it runs the computation again, we will need to pass  in `None` instead of `Some(hash)`.  Otherwise in `Pending` when it wakes up it could calculate the same hash value and skip the update.